README.md

```diff
@@ -1,2 +1,75 @@
 # Portfolio: setup and mobile version skeleton
- Module 1: Day 3 Task
+
+> Module 1: Day 3 Task
+To get familiarized with Flexbox, images, backgorunds and styling 
+
+
```

Comment:
- [ ] Kindly add the project title, `module 1: day 3 Task` is not a good title, also add a project description about what the project is about.